uPortSDK/Classes/Crypto/Crypto.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  Crypto.swift
+//  uPortSDK
+//
+
+import Foundation
+import Sodium
+
+/**
+ * This struct exposes methods to encrypt and decrypt messages according to the uPort spec at
+ * https://github.com/uport-project/specs/blob/develop/messages/encryption.md
+ */
+public struct Crypto {
+    private static let ASYNC_ENC_ALGORITHM = "x25519-xsalsa20-poly1305"
```

Comment:
In iOS constants (since Swift 3) are formatted `asyncEncAlgorithm`. They are also often placed/grouped in a `struct`:

```
struct Constants
{
    private static let asyncEncAlgorithm = "x25519-xsalsa20-poly1305"
    public static let blockSize = 64
}
```